### ISSUE TITLE
Failed to open the source

### ISSUE THREAD

**Issue by n-gao** [microsoft/winget-cli#998]
# Brief description of your issue

Calling any winget command results in an error message.
```
(base) ~> winget list
Failed to open the source; try the 'source reset' command if the problem persists.
An unexpected error occurred while executing the command:
0x8a15000f : Data required by the source is missing
```
# Steps to reproduce
Entering any command like `winget list` or `winget search`.

# Expected behavior
The command should execute without an error message

# Actual behavior

```
(base) ~> winget list
Failed to open the source; try the 'source reset' command if the problem persists.
An unexpected error occurred while executing the command:
0x8a15000f : Data required by the source is missing
```

# Environment

```none
[winget --info]
Windows Package Manager version: 0.4.11391 Preview
Windows: Windows.Desktop version v10.0.19042.985
Package: Microsoft.DesktopAppInstaller version v1.11.11391.0

Any other software?
```

**Comment by hitlerboy007** [comment]
The same bug;
When run 'winget list'，return '0x8a15000f : Data required by the source is missing'

**Comment by clovett** [comment]
I had the same problem after updating to windows 11, the following from an Admin command prompt solved the problem:

```
winget source reset winget
winget source add --name winget --arg https://winget.azureedge.net/cache --type Microsoft.PreIndexed.Package
```

Now winget install from non-admin prompt works fine again.

**Comment by denelon** [comment]
@n-gao

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).

Can you confirm if this is still happening?

**Comment by A9G-Data-Droid** [comment]
It is still happening
